templates/jaasai/big-data.html

```diff
@@ -17,7 +17,6 @@ <h2 class="p-heading--four" itemprop="description">The setup and configuration o
             url="https://assets.ubuntu.com/v1/9a372437-big-data-hero-image.svg",
             alt="",
             width="300",
-            height="108",
```

Comment:
Instead of removing the height lets define it correctly so that we don't get jumping content as the page loads the images.

Reply:
There are a few updates to the image library that we need to address so this branch can land with this change.